exercises/practice/quadratic-roots/quadratic_roots.py

```diff
@@ -0,0 +1,5 @@
+from z3 import *
+
+def roots(a: float, b: float, c: float) -> tuple:
+    """Solves for the two roots of the quadratic expression"""
```

Comment:
I would add a comment here indicating that this is where the user should add their code (you can use the same one that I have in mine to be consistent if you want).